osi_lane.proto

```diff
@@ -97,52 +112,15 @@ message Lane
         
         // A normal lane.
         // E.g. lanes with ids 1, 2, 3, 4 and 7 of the highway_exit image.
-        TYPE_NORMAL = 2;
```

Comment:
To retain backward compatiblity TYPE_NORMAL can be changed to TYPE_DRIVING, I think, but the new TYPE_NONDRIVING and TYPE_INTERSECTION enum values should be new unused values to avoid misinterpretation, I think...